aiohdfs/errors.py,sha256=EsBieTWwlz3sfLwtJtiF_I_5CjawrO8K3g6CSP9w1sU,370
aiohdfs/operations.py,sha256=XcByEY_9wFgDx3RCgCyKPvJiq3KEhLMi3A6EWGnZqiM,207
aiohdfs/__init__.py,sha256=TO6qeTnKEx7k6AMFdHKGmayIH7QQ2bRxIqy0YjdG7HQ,17468
aiohdfs-0.1.3.dev1.dist-info/DESCRIPTION.rst,sha256=WSLjvhvbPzvnci_a7J8NskQCQMfGDI3-Jvo6uuces8M,57
aiohdfs-0.1.3.dev1.dist-info/METADATA,sha256=6CmndR8V-y17Yyz6T0RNcSwLu9HtVlo566PXsovJn_Y,338
aiohdfs-0.1.3.dev1.dist-info/metadata.json,sha256=tSvx_JobN90XIA0ags4ENQtm_T4pfACVon3TY0KaNMk,559
aiohdfs-0.1.3.dev1.dist-info/pbr.json,sha256=UouFaaHrCEGgmv7zedCol0Z6hsMcrIzFaDKu7L6R9AU,47
aiohdfs-0.1.3.dev1.dist-info/RECORD,,
aiohdfs-0.1.3.dev1.dist-info/top_level.txt,sha256=XME6QoX5cBdzdAG_RMugNPogzM9mX7vmxSK0coxYkcQ,8
aiohdfs-0.1.3.dev1.dist-info/WHEEL,sha256=56WjMPRUEpmb30QaTzVbQCpri5lzKGby5HLDkAMQeIU,116
